pkg/plugin/istio/install.go

```diff
@@ -470,6 +425,9 @@ func (p *IstioPlugin) installRemotes(remotePilotAddress string) error {
 
 	for _, remote := range p.args.Remotes {
 		logrus.Infof("Begin to install istio in cluster %s", remote)
+		if err := waitSecertReady(p.Client, p.options, remote, caSecret); err != nil {
```

Comment:
why do we need to wait caCerts created?

Reply:
if istiod running before `istio-ca-certs` synced by `karmada`, istiod will self-signed a certs, member2 will fail to join.